cmd/sftp-server.go

```diff
@@ -197,6 +199,8 @@ func startSFTPServer(args []string) {
 			allowCiphers = filterAlgos(arg, strings.Split(tokens[1], ","), supportedCiphers)
 		case "mac-algos":
 			allowMACs = filterAlgos(arg, strings.Split(tokens[1], ","), supportedMACs)
+		case "user-ca-public-key":
```

Comment:
```suggestion
		case "trusted-user-ca-keys":
```

This is what it should be IMHO.